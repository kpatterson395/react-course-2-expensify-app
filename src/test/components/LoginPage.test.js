import React from 'react';
import { shallow } from 'enzyme';
import { ExpenseLoginPage } from '../../components/ExpenseLoginPage';

test('should correctly render ExpenseLoginPage', () => {
	const wrapper = shallow(<ExpenseLoginPage/>);
	expect(wrapper).toMatchSnapshot();
});

test('should call startLogin on button click', () => {
	const startLogin = jest.fn();
	const wrapper = shallow(<ExpenseLoginPage startLogin={startLogin} />);
	wrapper.find('button').simulate('click');
	expect(startLogin).toHaveBeenCalled();
});