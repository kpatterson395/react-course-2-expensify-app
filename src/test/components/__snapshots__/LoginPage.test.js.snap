// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render ExpenseLoginPage 1`] = `
<div>
  <button>
    Login
  </button>
</div>
`;
